src/pages/teamPage.tsx

```diff
@@ -1,44 +1,41 @@
-import React, { useState } from 'react';
+import React, { useEffect } from 'react';
 import { Spinner } from 'react-bootstrap';
 import { useDispatch, useSelector } from 'react-redux';
 import { useParams } from 'react-router-dom';
-import EmptyResult from '../components/emptyResult';
-import FetchError from '../components/fetchError';
-import SearchBar from '../components/searchBar';
-import SearchIcon from '../components/searchIcon';
-import TeamCard from '../components/teamCard';
+import SearchBar from '../components/SearchBar/searchBar';
+import SearchIcon from '../components/SearchBar/searchIcon';
 import { TEAM } from '../constants';
-import { toggleSearchBarActionCreator } from '../store/searchBar/searchBarActions';
 import { combinedStateInterface } from '../store/store';
+import { fetchDetailThunk } from '../services/api';
+import TeamInfo from '../components/TeamInfo/teamInfo';
+import FetchError from '../components/fetchError';
 
 interface urlParams {
-    Region: string;
-    Sport: string;
-    Club: string;
+    id: string;
 }
 
 const TeamPage = () => {
     const urlParams = useParams<urlParams>();
+    const dispatch = useDispatch();
     const reduxState = useSelector((state: combinedStateInterface) => state);
 
-    const listContent = reduxState.team.teams.map((entry) => {
-        return TeamCard({
-            id: entry.id,
-            name: entry.name,
-            full_capacity: entry.full_capacity,
-            short_description: entry.short_description,
-            long_description: entry.long_description,
-            tryouts: entry.tryouts,
-            registration_open: entry.registration_open,
-            group: entry.group,
-        });
+    useEffect(() => {
+        if (
+            !reduxState.thunk.fetch_in_progress &&
+            reduxState.thunk.fetch_failed_count < 3 &&
+            !reduxState.thunk.fetch_success
+        ) {
+            dispatch(fetchDetailThunk(TEAM, urlParams.id));
+        }
     });
 
+    const team = reduxState.team_detail.team;
+
     return (
         <div className="container">
             <div className="row">
                 <div className="col">
-                    <h1>{urlParams.Club}</h1>
+                    <h1>HEADER</h1>
```

Comment:
Assume this and some of the other headers are just temporary since this is just design prep and will get fixed when design is implemented?

Reply:
Yes, temporary before implementation of header is added